### PR TITLE
Added eslint rule import/order

### PR DESCRIPTION
**Pull request by priscilawebdev**
**Type:** Feat

**Description:**

In this PR, I am proposing to use the rule import/order to enforce a convention in module import order. You can read more about it here: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md

Please what do you think?

### CODE REVIEW

**Comment by sergiohgz** [comment]
```
@verdaccio/types:   1:1  error  Definition for rule 'import/order' was not found  import/order
```
The rule is not defined 🤔 take a look if something is not loading correctly

**Comment by priscilawebdev** [comment]
Hmmm now the error is: 
image

File: tools/eslint-config/index.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ module.exports = {
     {
       files: ['*.ts', '*.tsx'],
       extends: ['./rules/base', './rules/typescript', './rules/import', './rules/prettier'].map(require.resolve),
+      plugins: ['import'],
```
Comment (sergiohgz) [inline tools/eslint-config/index.js]: This line should be moved to `rules/import.js` file (I tested it)

Comment (priscilawebdev) [reply tools/eslint-config/index.js]: I see...thank you!! :-)

**Pull request merged**